src/bundle/Resources/views/content/tab/urls.html.twig

```diff
@@ -57,19 +57,34 @@
     }) }}
 {% endif %}
 
-{% include '@ezdesign/content/tab/url/modal_add_custom_url.html.twig' with {'form': form_custom_url_add, 'parent_name': parent_name} only %}
+{% if can_edit_custom_url %}
+    {% include '@ezdesign/content/tab/url/modal_add_custom_url.html.twig' with {
+        'form': form_custom_url_add,
+        'parent_name': parent_name
+    } only %}
+{% endif %}
 
 {% macro table_header_tools(form_custom_url_remove, can_edit_custom_url) %}
     {% if can_edit_custom_url %}
-        <button class="btn btn-primary ez-btn--prevented" data-toggle="modal" data-target="#ez-modal--custom-url-alias" title="{{ 'tab.urls.action.add'|trans|desc('Add Custom URL') }}">
+        <button
+            class="btn btn-primary ez-btn--prevented"
+            data-toggle="modal"
+            data-target="#ez-modal--custom-url-alias"
```

Comment:
That's weird `id`. The BEM naming convention applies only to CSS class names, not to `id` attributes.

Reply:
same here